django/contrib/admin/options.py

```diff
@@ -1754,8 +1770,9 @@ def _delete_view(self, request, object_id, extra_context):
 
         # Populate deleted_objects, a data structure of all related objects that
         # will also be deleted.
-        (deleted_objects, model_count, perms_needed, protected) = get_deleted_objects(
-            [obj], opts, request.user, self.admin_site, using)
+        deleted_objects, model_count, perms_needed, protected = self.get_deleted_objects(
```

Comment:
I think a test for this change is missing (similar to `test_overriding_get_deleted_objects`). This would probably go in `admin_views` whereever the other tests for the `delete_view` are.